loader/loader.go

```diff
@@ -136,6 +137,13 @@ func (w *Worker) run(ctx context.Context, fileJobQueue chan *fileJob, workerWg *
 				offsetSQL := w.checkPoint.GenSQL(job.file, job.offset)
 				sqls = append(sqls, offsetSQL)
 
+				failpoint.Inject("LoadExceedOffsetExit", func(val failpoint.Value) {
+					threshold, _ := val.(int)
+					if job.offset >= int64(threshold) {
+						panic(fmt.Sprintf("load offset %d execeeds threshold %d", job.offset, threshold))
```

Comment:
should we include any marker (like `[failpoint]`) in the log/message triggered by failpoint? so we can diff them from normal log/message. see a panic message may make us panic until we search it in the codebase.

Reply:
addressed